src/main/java/com/akolb/HMSBenchmarks.java

```diff
@@ -285,6 +285,56 @@ static DescriptiveStatistics benchmarkGetPartitionsByName(MicroBenchmark bench,
     }
   }
 
+  static DescriptiveStatistics benchmarkRenameTable(MicroBenchmark bench,
+                                                    final HMSClient client,
+                                                    final String dbName,
+                                                    final String tableName,
+                                                    final String newTableName,
+                                                    int count) {
+    createPartitionedTable(client, dbName, tableName);
+    try {
+      addManyPartitionsNoException(client, dbName, tableName,
+          Collections.singletonList("d"), count);
+      Table oldTable = client.getTable(dbName, tableName);
+      oldTable.getSd().setLocation("");
+      Table newTable = oldTable.deepCopy();
+      newTable.setTableName(newTableName);
+
+      return bench.measure(
+          null,
+          () -> {
+              client.alterTableNoException(oldTable.getDbName(), oldTable.getTableName(), newTable);
```

Comment:
This is an interesting point. If you don't have pre and post-run, you can just use a single-argument variation of measure().

Essentially you are measuring time to do two renames - right? In this case this seems ok.